skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -152,11 +147,8 @@ private function getCuratedContentButton() {
 
 	/**
 	 * Render default page header (with edit dropdown, history dropdown, ...)
-	 *
-	 * @param: array $params
-	 *    key: showSearchBox (default: false)
 	 */
-	public function executeIndex( $params ) {
+	public function index() {
 		wfProfileIn( __METHOD__ );
 
 		$this->isUserLoggedIn = $this->wg->User->isLoggedIn();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)